ansible/playbooks/nagios/documentation/USAGE_GUIDE.md

```diff
@@ -0,0 +1,62 @@
+
+**Usage Guide :**
+
+This document will cover a number of installation scenarios, firstly a standard installation on a typical Linux (Ubuntu) server, and secondly a set of instructions to cover creating a development or test server using vagrant.
+
+Installing On A Linux Server
+---------------------------------------
+
+Ansible is required to run the playbooks, either directly on the Nagios server host, or alternatively from an Ansible machine with the ability to connect to the Nagios server. The playbooks will need to run as the root user, as it will need to install a number of packages, and create a "nagios" user, for the server to run as.
+
+1) Prior to running the Nagios server installation playbook, ensure the ansible.cfg, nagios_inventory and the secrets_setup_server.enc vault file have been updated as necessary. This is detailed in the main README.MD file.
+
+2) To run the first playbook (which will create a base nagios server installation)
+
+    ansible-playbook -b play_setup_server.yml --ask-vault-pass  
+
+3) Once this is completed, the Nagios server will be accessible via the Web UI, (access via http://nagios.server.id/nagios), replacing the nagios.server.id with the IP address or hostname for the server Nagios has been installed on.
```

Comment:
Reads as a generic solution - use actual hostnames (or do you prefer not to do so)?